autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -79,9 +82,55 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Note: final model returned may be fit on this tuning_data as well as train_data. Do not provide your evaluation test data here!
             In particular, when `num_bagging_folds` > 0 or `stack_ensemble_levels` > 0, models will be trained on both `tuning_data` and `train_data`.
             If `tuning_data = None`, `fit()` will automatically hold out some random validation examples from `train_data`.
-        output_directory : str
+        presets : list or str or dict, default = 'medium_quality_fastest_train'
+            List of preset parameter configurations.
+            It is recommended to specify presets and avoid specifying most other parameters prior to becoming familiar with AutoGluon.
+            As an example, to get the best model quality, set `presets='best_quality'`.
+            To get good quality with minimal disk usage, set `presets=['good_quality_faster_inference_only_refit', 'optimize_for_deployment']`
+            User specified parameters will override changes made by presets.
+            If specifying a list of presets, later presets will override earlier presets if they alter the same parameter.
+            Users can specify custom presets by passing in a dictionary of parameter values.
+
+            Available Presets: ['best_quality', 'best_quality_with_high_quality_refit', 'high_quality_fast_inference_only_refit', 'good_quality_faster_inference_only_refit', 'medium_quality_fastest_train', 'optimize_for_deployment', 'ignore_text']
```

Comment:
clarify which available presets are compatible with each other vs. not

Reply:
Added